### PR TITLE
Refactor packages

### PR DESCRIPTION
**Pull request by jalvz**
depends on #102

### CODE REVIEW

File: worker/work.go [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+	err := w.Run()
+	report.End = time.Now()
+	w.flush()
+	report.Flushed = time.Now()
+
+	rs := w.Stats()
+	report.add("transactions sent", rs.TransactionsSent)
+	report.add("transactions dropped", rs.TransactionsDropped)
+	if rs.TransactionsSent+rs.TransactionsDropped > 0 {
+		report.add("transactions sent Perct.", numbers.Perct(rs.TransactionsSent, rs.TransactionsDropped))
```
Comment (graphaelli) [inline worker/work.go]: maybe `success %` ?

File: worker/work.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+		report.add("spans sent", rs.SpansSent)
+		report.add("spans dropped", rs.SpansDropped)
+		report.add("spans sent Perct.", numbers.Perct(rs.SpansSent, rs.SpansDropped))
```
Comment (graphaelli) [inline worker/work.go]: same idea, `success %` ?

File: worker/work.go [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+		}
+	}
+	report.add("errors sent", rs.ErrorsSent)
+	report.add("errors dropped", rs.ErrorsDropped)
+	if rs.ErrorsSent+rs.ErrorsDropped > 0 {
+		report.add("errors sent Perct.", numbers.Perct(rs.ErrorsSent, rs.ErrorsDropped))
+	}
+	report.add("failed", rs.Errors.SendStream)
+	eventsSent := float64(rs.ErrorsSent + rs.SpansSent + rs.TransactionsSent)
+	report.add("events sent per second", eventsSent/report.Flushed.Sub(report.Start).Seconds())
```
Comment (graphaelli) [inline worker/work.go]: worth it to include total events sent too?  Maybe combine with the per second line like `events sent .... 123 (12.5/s)`

Comment (jalvz) [reply worker/work.go]: done in https://github.com/elastic/hey-apm/pull/102/commits/714ac272827a7cc1bfb467dd8f3f9b4dcdb2d70f

(i never know when or not to split up PRs...)

**Pull request merged**